study.md

```diff
@@ -137,7 +143,8 @@ Using JavaScript's Array methods, set a variable named `hankKeys` to `hank`'s
 keys.  Additionally, set a variable named `hankValues` to `hanks`'s values.'
 
 ```javascript
-// your answer here
+let hanksValues = hank.values()
```

Comment:
`values()` and `keys()` are not methods on `hank`.  You need to get the values using `Object.values()` and the keys using `Object.keys()`, a loop, or an iteration method.